app/src/main/java/org/schabi/newpipe/RouterActivity.java

```diff
@@ -126,6 +127,15 @@ protected void onCreate(final Bundle savedInstanceState) {
                 ? R.style.RouterActivityThemeLight : R.style.RouterActivityThemeDark);
     }
 
+    @Override
+    protected void onStop() {
```

Comment:
is there any reason why you decided to use onStop in favor of onPause? I think it could also be put there to be cleared up as early as possible

Reply:
I don't know, I don't think it matters much

Reply:
the reason is as @Stypox stated.